internal/cli/apps.go

```diff
@@ -868,48 +868,37 @@ func stringSliceToPtr(s []string) *[]string {
 	return &s
 }
 
-func (c *cli) appPickerOptions() (pickerOptions, error) {
-	list, err := c.api.Client.List()
-	if err != nil {
-		return nil, err
-	}
-
-	tenant, err := c.getTenant()
-	if err != nil {
-		return nil, err
-	}
-
-	// NOTE(cyx): To keep the contract for this simple, we'll rely on the
-	// implicit knowledge that the default value for the picker is the
-	// first option. With that in mind, we'll use the state in
-	// tenant.DefaultAppID to determine which should be chosen as the
-	// default.
-	var (
-		priorityOpts, opts pickerOptions
-	)
-	for _, c := range list.Clients {
-		// Empty type means the default client that we shouldn't display.
-		// TODO(sergiught): We only need to exclude generic app types for
-		// the auth0 qs download command. Fix this in separate PR.
-		if c.GetAppType() == "" {
-			continue
+func (c *cli) appPickerOptions(requestOpts ...management.RequestOption) pickerOptionsFunc {
+	requestOpts = append(requestOpts, management.Parameter("is_global", "false"))
```

Comment:
After this PR we can follow up with another one where we can remove the "All Applications" hack removal from the apps display pkg to skip showing the global client.